### ISSUE TITLE
Oddly Shaped Asteroids and Regenerating only when in suit.

### ISSUE THREAD

**Issue by Murllin** [Equinox-/SEProceduralWorld#12]
I've seen some of the asteroids spawn as a small "cube" shape, however only 3 sides will be visible. I only know its a cube because I've flown to them in my suit and I hit the other 3 invisible sides.

Also I've noticed that the asteroids will only generate for people if they are out in their space suit. If some one happens across the asteroid field while piloting a ship, the asteroids do not generate. They have to actually stop and hop out for a second to get the asteroids to spawn.

**Comment by Geneticus** [comment]
What you are describing is a visual bug in how LODs are generated. A very small object will be deleted as a fine detail from the renderer. It “pops” into existence” when the camera gets close to it.

**Comment by Murllin** [comment]
Ok so is there a fix for it? If some one one approaches it from the invisible side with their ship it will destroy them. Its kind of a nousance.

**Comment by Geneticus** [comment]
Yes, it’s a very old nuisance. From what Equinox has told me, all he is doing is guessing at what asteroids the world generator will create and blocking the ones out that hit his filter.  He might be able to filter out asteroids with straggling voxels, I can’t say for sure.
